client/app/controllers/projects.js

```diff
@@ -31,18 +36,8 @@ export default class ProjectsController extends Controller {
     return this.model.filter((project) => {
       // if pasPackages is empty, some() automatically returnse false, but we want to return true.
       if (project.pasPackages.length === 0) return true;
-      return project.pasPackages.some((projectPackage) => {
-        if (
-          projectPackage.statuscode === PACKAGE_STATUS.PACKAGE_PREPARATION.code
-          && [
-            PACKAGE_VISIBILITY.APPLICANT_ONLY.code,
-            PACKAGE_VISIBILITY.GENERAL_PUBLIC.code,
-          ].includes(projectPackage.dcpVisibility)
-        ) {
-          return false;
-        }
-        return true;
-      });
+      // Check that NONE of the packages are currently editable
+      return !packageIsToDo(project.pasPackages) && !packageIsToDo(project.rwcdsPackages);
```

Comment:
We can just negate the `toDoProjects` getter to compute this getter. e.g.
```
get doneProjects() {
  return this.model.filter((project) => {
      return !this.toDoProjects.includes(project);
    })
}
```

Reply:
Yeah that works!